charts/infra-apps/values.yaml

```diff
@@ -196,3 +196,23 @@ kured:
   # kured.values -- Helm values
   # @default -- [upstream values](https://github.com/weaveworks/kured/blob/main/charts/kured/values.yaml)
   values: {}
+
+# kubeEventExporter -- [kubernetes-event-exporter](https://github.com/opsgenie/kubernetes-event-exporter)
+# @default -- [example](./examples/kubernetes-event-exporter.yaml)
+kubeEventExporter:
+  # kubeEventExporter.enabled -- Enable kubernetes-event-exporter
+  enabled: false
+  name: event-exporter
+  destination:
+    # kubeEventExporter.destination.namespace -- Namespace
+    namespace: infra-event-exporter
```

Comment:
shouldn't we be more precise in calling this `infra-kube-event-exporter`. Just `event-exporter` isn't clear for most if it's regarding k8s or something else?

Reply:
i was hedging my hate of long names but will make it consistent and clear

Reply:
it is now  kube-event-exporter all the way through, no more plain event-exporter

Reply:
I hate long names too.